tests/test_benchmark_cli.py

```diff
@@ -218,3 +219,16 @@ def test_format_conversion_ion_text_to_ion_binary():
     rewrite_file_to_format(generate_test_path('integers.10n'), Format.Format.ION_TEXT.value)
     assert os.path.exists('temp_integers.ion')
     os.remove('temp_integers.ion')
+
+
+@parametrize(
+    ('write', 'json', generate_test_path('./sample_spec/multiple_top_level_object.json')),
+    ('write', 'cbor2', generate_test_path('./sample_spec/multiple_top_level_object.cbor')),
+    ('read', 'json', generate_test_path('./sample_spec/multiple_top_level_object.json')),
+    ('read', 'cbor2', generate_test_path('./sample_spec/multiple_top_level_object.cbor')),
+)
+def test_multiple_top_level_values(args):
+    (command, format_option, file) = args
+    (error_code, _, _) = run_cli([f'{command}', file, '--format', f'{format_option}', '--io-type', 'file'])
+    assert not error_code
```

Comment:
Are we able to assert that multiple top-level values are actually read/written?

Reply:
As discussed offline, I'll create a few new tests for each generated test_fun to ensure they work correctly.